core/src/main/scala/cats/data/XorT.scala

```diff
@@ -51,6 +51,8 @@ final case class XorT[F[_], A, B](value: F[A Xor B]) {
 
   def exists(f: B => Boolean)(implicit F: Functor[F]): F[Boolean] = F.map(value)(_.exists(f))
 
+  def ensure[AA >: A](ifLeft: => AA)(f: B => Boolean)(implicit F: Functor[F]): XorT[F, AA, B] = XorT(F.map(value)(_.ensure(ifLeft)(f)))
```

Comment:
Well, in case of predicate falling the whole `XorT` "falls" to the left branch, it might be an explanation :)

But no problem, I could rename both.